### PR TITLE
Implement offline gRPC bridge.

### PR DESCRIPTION
**Pull request by cschuet**

### CODE REVIEW

File: cartographer_ros/cartographer_ros/cartographer_grpc/offline_node_grpc_main.cc [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  google::InitGoogleLogging(argv[0]);
+  google::ParseCommandLineFlags(&argc, &argv, true);
+
+  CHECK(!FLAGS_configuration_directory.empty())
+      << "-configuration_directory is missing.";
+  CHECK(!FLAGS_configuration_basename.empty())
+      << "-configuration_basename is missing.";
+  CHECK(!FLAGS_bag_filenames.empty()) << "-bag_filenames is missing.";
+
+  ::ros::init(argc, argv, "cartographer_offline_node");
```
Comment (gaschler) [inline cartographer_ros/cartographer_ros/cartographer_grpc/offline_node_grpc_main.cc]: Correct this to the executable name

Comment (cschuet) [reply cartographer_ros/cartographer_ros/cartographer_grpc/offline_node_grpc_main.cc]: Done. Thanks for catching this!

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

**Pull request merged**